app/models/shelter.rb

```diff
@@ -1,3 +1,4 @@
 class Shelter < ApplicationRecord
-  validates_presence_of :name, :address, :city, :state, :zip
+  has_many :pets
+  validates_presence_of :name
```

Comment:
I changed this because I feel the shelter name is the only thing completely necessary, but will reevaluate down the line